composer.json

```diff
@@ -3,12 +3,13 @@
     "description": "External module for generating report counts.",
     "type": "project",
     "scripts": {
-      "test": "phpunit --colors=always",
-      "test-jenkins": "phpunit --colors=never --log-junit junit/phpunit.xml"
+      "test": "vendor/bin/phpunit --colors=always",
+      "test-jenkins": "vendor/bin/phpunit --colors=never --log-junit junit/phpunit.xml"
```

Comment:
Without specifying the `vendor/bin` directory, whatever version of PHPUnit is on the path is used, which fails to find the dependencies, which causes the tests to fail.